### PR TITLE
Rspec Testing

### PR DESCRIPTION
**Pull request by rayking12**
Things to look at for:
tests for each method in the tic tac toe project

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @rayking12 :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this milestone can be approved.

## REQUIRED

- [x] Fix linter errors
- [x] Specs for each method should be wrapped in one describe block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples . (screenshot from Pathwright)
- [ ] Your specs should cover positive and negative scenarios ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)) Example: there is a spec for correct usage of the method \[e.g. `it "creates object if all required attributes are passed"`\] and also there is spec for the wrong usage of a method \[e.g. `it "does not create the object if required attributes are missing"`\].
## Make the required changes and submit them for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 

Great job @team :clap: :clap: :clap: :clap: :clap:, thanks for making the required :+1: :100: changes.

Now you can merge, your project is approved.

## Optional:
- [ ] Add more tests if you can, so you can practice more with unit tests.

## Happy Coding :tophat:

**Pull request closed without merging**
